TeachingRecordSystem/src/TeachingRecordSystem.SupportUi/Pages/RoutesToProfessionalStatus/EditRoute/EditRouteStatusState.cs

```diff
@@ -4,7 +4,6 @@ public class EditRouteStatusState
 {
     public RouteToProfessionalStatusStatus Status { get; set; }
     public DateOnly? HoldsFrom { get; set; }
-    public DateOnly? TrainingEndDate { get; set; }
     public bool? InductionExemption { get; set; }
     public bool RouteImplicitExemption { get; init; }
 }
```

Comment:
this is the state used to store the collection of data when moving to awarded - end date has been removed from that data collection